Lib/zipfile.py

```diff
@@ -957,6 +968,51 @@ def close(self):
         finally:
             super().close()
 
+    def seekable(self):
+        return self._seekable
+
+    def seek(self, offset, from_what = 0):
```

Comment:
I would recommend matching the signature of the same method in the `io` module: `seek(self, pos, whence=0)`

Reply:
I chose offset because I can better differentiate between offset value (relative values) and position values (absolute values). Also, according to the official documentation "offset" should be the first argument for seek (although there is no consensus when I search through the cpython source). https://docs.python.org/3/library/io.html#io.IOBase.seek

I agree with changing "from_what" to "whence". The former always felt awkward to me. Thank you for the suggestion

Reply:
It seems like older modules use `pos` (e.g. mmap), and the more recent io ABCs use `offset`.  This should be fine, and I expect people to pass the parameter as a positional argument anyway.

Reply:
All methods in the stlib use `whence` though, I would keep that.